tests/test_system.py

```diff
@@ -18,6 +18,49 @@ def test_version_endpoint(client):
     assert version
 
 
+def test_schema_endpoint_basic(client):
```

Comment:
could you add a unit test that checks the output of `_pretty_type` specifically, to ensure that the `type` generated is what we expect for all 3 supported input formats